backend/game/player/player.py

```diff
@@ -32,3 +36,9 @@ def update_energy_by(self, change_integer):
         self.energy += change_integer
         if self.energy < constants.DEFAULT_ENERGY_CUBE:
             self.energy = constants.DEFAULT_ENERGY_CUBE
+
+    def add_keep_card(self, card):
+        self.card_on_hand[card.name] = card.effect
```

Comment:
I do not see how this is helpful or necessary, the effect is a string representation of what the card should do,